Jenkinsfile

```diff
@@ -1,24 +1,30 @@
 #!groovy
 
-def supportedPhpVersions = ["5.6", "7.0", "7.1"]
+String launchUnitTests = "yes"
+String launchIntegrationTests = "yes"
+String pimVersion = "1.7"
 
-def launchUnitTests = "yes"
-def launchIntegrationTests = "no"
-def clients = ["php-http/guzzle5-adapter", "php-http/guzzle6-adapter"]
-// def pimVersion = "1.7"
+def clientConfig = [
+    "php-http/guzzle6-adapter": ["phpVersion": ["5.6", "7.0", "7.1"], "psrImplem": ["guzzlehttp/psr7"]],
+    "php-http/guzzle5-adapter": ["phpVersion": ["5.6", "7.0", "7.1"], "psrImplem": ["guzzlehttp/psr7", "zendframework/zend-diactoros", "slim/slim"]],
+    "php-http/curl-client": ["phpVersion": ["5.6", "7.0", "7.1"], "psrImplem": ["guzzlehttp/psr7", "zendframework/zend-diactoros", "slim/slim"]]
+    //"php-http/socket-client": ["phpVersion": ["5.6", "7.0", "7.1"], "psrImplem": ["guzzlehttp/psr7", "zendframework/zend-diactoros", "slim/slim"]]
```

Comment:
don't merge commented code :P it's a debt you will forget soon, better remove it